app/src/main/java/com/unithon/ott/presentation/ui/album/AlbumActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.unithon.ott.presentation.ui.album
+
+import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
+import com.unithon.ott.R
+import com.unithon.ott.common.base.BaseActivity
+import com.unithon.ott.databinding.ActivityAlbumBinding
+
+class AlbumActivity : BaseActivity<ActivityAlbumBinding>(R.layout.activity_album) {
+
+    // 가족 사진관
+
+    private val viewModel by lazy {
+        ViewModelProvider(
+            this,
+            ViewModelProvider.NewInstanceFactory()
+        )[AlbumViewModel::class.java]
+    }
```

Comment:
```suggestion
     private val albumViewModel : AlbumViewModel by viewModels()
```
뷰모델에 파라메터값 전달되는 거 없어서 바로 이렇게 사용하는게 좋을 것 같아요